darts/models/rnn_model.py

```diff
@@ -96,6 +96,9 @@ def forward(self, x, h=None):
 
 
 class RNNModel(TorchParametricProbabilisticForecastingModel, DualCovariatesTorchModel):
+
+    uses_past_covariates = False
```

Comment:
Note to self: search for other models like this. @hrzn is there a fast way to find models, which support both types of covariates in fit(), but they use only one?

Reply:
Here's the list:
* `PastCovariatesTorchModel`: only past
* `FutureCovariatesTorchModel`, `DualCovariatesTorchModel` and `DualCovariatesForecastingModel`: only future
* `MixedCovariatesTorchModel` and `SplitCovariatesTorchModel`: both

Reply:
cool, thanks!

Reply:
done.